Docs: Improve API tutorial intro content and readability

kumaraguru-ramasamy opened the pull request:
- Simplified the introductory paragraphs.
- Rewrote the descriptions and steps to improve readability.

<!--

Thank you for sending a pull request! Here are some tips:

1. If this is your first time, please read our contribution guide at https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md

2. Ensure you include and run the appropriate tests as part of your Pull Request.

3. In a new feature or configuration option, an update to the documentation is necessary. Everything related to the documentation is under the docs folder in the root of the repository.

4. If the Pull Request is a work in progress, make use of GitHub's "Draft PR" feature and mark it as such.

5. If you can not merge your Pull Request due to a merge conflict, Rebase it. This gets it in sync with the master branch.

6. Name your PR as "<FeatureArea>: Describe your change", e.g. Alerting: Prevent race condition. If it's a fix or feature relevant for the changelog describe the user impact in the title. The PR title is used to auto-generate the changelog for issues marked with the "add to changelog" label.

-->

**What this PR does / why we need it**: The introductory paragraphs are not clear to get started with authentication methods and API tokens. Reviewed the content for clarity and proposed edits.

**Which issue(s) this PR fixes**: None

<!--

* Automatically closes linked issue when the Pull Request is merged.

Usage: "Fixes #<issue number>", or "Fixes (paste link of issue)"

-->

Fixes #

**Special notes for your reviewer**: Please review the proposed changes. The topic is edited to improve clarity and readability.

kumaraguru-ramasamy committed:
API Tutorial: Improve content and readability

- Simplified the introductory paragraphs.
- Rewrote the descriptions and steps to improve readability.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=18762) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=18762) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=18762) it.</sub>

marcusolsson commented on docs/sources/tutorials/api_org_token_howto.md:
```diff
@@ -9,13 +9,14 @@ weight = 10
+There are two authentication methods to access the API: 
 
-Some parts of the API are only available through basic authentication and these parts of the API usually require that the user is a Grafana Admin. But all organization actions are accessed via an API Token. An API Token is tied to an organization and can be used to create dashboards etc but only for that organization.
+- Basic authentication: A Grafana Admin user can access some parts of the Graphana API through basic authentication. 
```
"Graphana" -> "Grafana"

kumaraguru-ramasamy replied:
Looks good, Marcus!

I was browsing through the Grafana website and the graphs on the Home page just occupied my mind!

marcusolsson commented on docs/sources/tutorials/api_org_token_howto.md:
````diff
@@ -71,4 +72,4 @@ Using the Token that was created in the previous step, you can create a dashboar
   ```
 
-  This import will not work if you exported the dashboard via the Share -> Export menu in the Grafana UI (it strips out data source names etc.). View the JSON and save it to a file instead or fetch the dashboard JSON via the API.
+  This import will not work if you export the dashboard using the Share -> Export menu in the Grafana UI (it strips out data source names etc.). View the JSON and save it to a file instead or fetch the dashboard JSON through the API.
````
This is an improvement, but I'm thinking we can do even more with this paragraph. 

Maybe something like this?

_Note that it is not possible to import a dashboard that was exported using the Share -> Export menu in the Grafana UI. Instead, either select View JSON, and save it to a file, or fetch the JSON through the API._

marefr replied:
Actually, since an update while ago in Share -> Export you have to actively select that you want to export a templated dashboard, otherwise it will behave the same as "View JSON", for reference:
![image](https://user-images.githubusercontent.com/1668778/63943911-daf2bd80-ca70-11e9-8dc8-c98a9a8613e5.png)

So maybe we could improve this some more by referring to " import a dashboard that was exported to be shared externally" or something similar?

kumaraguru-ramasamy replied:
Updated the note as suggested by reviewers.

marcusolsson reviewed (changes requested):
Thank you for this! I think we could merge this as is (apart from a typo). I made a suggestion on the last paragraph though. Let me know what you think!

kumaraguru-ramasamy committed:
Updated topic as discussed in the PR

- Corrected the typo from "Graphana" to "Grafana".
- Rewritten the note as suggested by reviewers and to avoid passive voice.

Please review and share your thoughts.
